packages/web/src/components/user-list/UserList.tsx

```diff
@@ -99,7 +99,8 @@ export const UserList = ({
     if (isMobile) {
       dispatch(unfollowConfirmationActions.setOpen(userId))
     } else {
-      dispatch(socialActions.unfollowUser(userId, FollowSource.USER_LIST))
+      // TODO: this will be fixed in a follow up pr addressing user-lists
+      // dispatch(socialActions.unfollowUser(userId, FollowSource.USER_LIST))
```

Comment:
user lists are migrated so can we address this now?

Reply:
not yet, that is still open actually